packages/tupaia-web/src/features/PDFExportDashboardItem/PDFExportDashboardItem.tsx

```diff
@@ -21,6 +21,14 @@ import { useReport } from '../../api/queries';
 import { DashboardItemContent, DashboardItemContext } from '../DashboardItem';
 import { PDFExportHeader } from './PDFExportHeader';
 
+const StyledA4Page = styled(A4Page)<{
+  $isPreview?: boolean;
+}>`
+  ${({ $isPreview }) => ($isPreview ? 'width: 100%;' : '')};
```

Comment:
If you wanted to you could probably group the preview styles together instead of doing multiple ternaries